src/ILRepack.props

```diff
@@ -0,0 +1,27 @@
+<?xml version="1.0" encoding="utf-8" ?>
```

Comment:
Any particular reason the props and targets file need to be separated? I don't see any projects using any of the properties, and this doesn't really need to be overly generic since it's just used within the repo.

Reply:
I'll take a look- it might be a vestige of using the ILRepack Task